tests/js/jest.config.js

```diff
@@ -37,7 +37,12 @@ module.exports = {
 	// Matches aliases in webpack.config.js.
 	moduleNameMapper: {
 		// New (JSR) modules.
-		'^googlesitekit-(.+)$': '<rootDir>assets/js/googlesitekit-$1',
```

Comment:
Mapped each module separately now that we have modules with varying file names (e.g. `-gm2.js`).